src/Angor/Client/Storage/ClientStorage.cs

```diff
@@ -85,6 +85,18 @@ public void UpdateInvestmentProject(ProjectInfo project)
         _storage.SetItem("projects", ret);
     }
 
+    public void RemoveInvestmentProject(string projectId)
+    {
+        var ret = GetInvestmentProjects();
+
+        var item = ret.First(_ => _.ProjectIdentifier == projectId);
+
+        if (!ret.Remove(item))
+            throw new InvalidOperationException();
```

Comment:
Do we actually need the exception? if it's not in the list there's nothing to remove.

Reply:
good point I will fix